vite.config.ts

```diff
@@ -0,0 +1,94 @@
+import { extname } from "path";
+import { defineConfig, UserConfig } from "vite";
+import viteCompression from "vite-plugin-compression";
+import { injectCanisterIdPlugin, stripInjectJsScript } from "./vite.plugins";
+
+const defaultConfig = (mode?: string): Omit<UserConfig, "root"> => {
+  // Path "../../" have to be expressed relative to the "root".
+  // e.g.
+  // root = src/frontend
+  // envDiv = ../../ because .env is at the base of the project
+
+  return {
+    envDir: "../../",
```

Comment:
@peterpeterparker can you confirm that you use this locally?

Reply:
Removed.